### DISCUSSION TITLE
[Projects beta] Allow other date formats

### DISCUSSION THREAD

**Discussion by danieleldjarn** [community/community#7269] [Projects and Issues]
When creating a column of the type _date_ it shows up using US date notion. It would we nice to allow other formats e.g. YYYY-MM-DD

![image](https://user-images.githubusercontent.com/806188/140307763-7b97f0d4-5b7a-4fb7-8ad1-a5f5b9f054a5.png)

**Comment by srittau** [comment]
Especially since the US notation is one of the worst possible notations for use in tables. Same goes for month names, instead of ordinal months. It makes it hard to understand the relations of different dates in a column to each other.

**Comment by Relequestual** [comment]
Our project works across all timezones. We would love to standardise our dates as ISO, aka YYYY-MM-DD.
